Updates to Nutraceuticals (#230)
greenelab/covid19-review: content/20.treatments.md

A recent meta-analysis found consistent support for regular vitamin C supplementation reducing the duration of the common cold, but that supplementation with vitamin C (> 200 mg) failed to reduce the incidence of colds [@doi:10.1002/14651858.CD000980.pub4].
Individual studies have found Vitamin C to reduce the susceptibility of patients to lower respiratory tract infections such as pneumonia [@doi:10.1097/00006454-199709000-00003].
Another meta-analysis has demonstrated in twelve trials that vitamin C supplementation reduced the length of stay of patients in intensive care units (ICUs) by 7.8% (95% CI: 4.2% to 11.2%; p = 0.00003).
Furthermore, high doses (1-3 g/day) significantly reduced the length of an ICU stay by 8.6% in six trials (p = 0.003).

> Vitamin C also shortened the duration of mechanical ventilation by 18.2% in three trials in which patients required intervention for over 24 hours (95% CI 7.7% to 27%; p = 0.001) [@doi:10.3390/nu11040708].

Reviewer: Belated comment, but perhaps it would be good to specify more exactly what you mean by vitamin C since there are a few different ways these studies go (could be "not being Vitamin C deficient", normal dosage, high dosage)

Author: Hey @rando2 The reason I haven't been specific is that there is difficulty in conducting meta-analyses of supplements as various doses are used, some studies account for deficiency, some don't. There are a lot of issues, which the meta-analyses themselves comment on too. 

Do you think a sentence on that is warranted, or just provide more detail?

Thanks!

Reviewer: Oh, if it's intentional, then that's totally fine with me! It seems like such an interesting body of literature, there is a lot going on!

Author: Its continuing to grow, I have to add covid trials to the zinc and vit D section haha

Despite these findings, the CITRUS ALI study failed to show a benefit of a 96-hour infusion of vitamin C to treat ARDS, which is a severe complication of COVID-19 infection [@doi:10.1001/jama.2019.11825].

> Nevertheless, a randomized placebo-controlled trial [@url:https://clinicaltrials.gov/ct2/show/NCT04264533] has begun in Wuhan China to investigate the intravenous infusion of vitamin C to treat pneumonia in 140 severe COVID-19 infected patients. 

Reviewer: ```suggestion
Nevertheless, a randomized placebo-controlled trial [@url:https://clinicaltrials.gov/ct2/show/NCT04264533] has begun in Wuhan, China to investigate the intravenous infusion of vitamin C to treat pneumonia in 140 severe COVID-19 infected patients. 
```

> Nevertheless, a randomized placebo-controlled trial registered with clincialtrials.gov (Identifier:NCT04264533), has begun in Wuhan China to investigate the intravenous infusion of vitamin C to treat pneumonia in 140 severe COVID-19 infected patients. 

Author: Thank you both for fixing!

As summarized by Carr [@doi:10.1186/s13054-020-02851-4] the trial will not be completed until September 2020.

> Another trial in Italy registered with clincialtrials.gov (Identifier: NCT04323514) intends to deliver a 10 g infusion of vitamin C to 500 severe COVID-19 patients with pneumonia to assess in-hospital mortality over a 72 hr period as the primary outcome. 

Reviewer 2: ```suggestion
Another trial in Italy [@url:https://clinicaltrials.gov/ct2/show/NCT04323514] intends to deliver a 10 g infusion of vitamin C to 500 severe COVID-19 patients with pneumonia to assess in-hospital mortality over a 72 hr period as the primary outcome. 
```

The trial is currently recruiting and is due to run until March 2021.

> Unfortunately, it will not be known how effective vitamin C is as a thereapeutic for quite some time due to the length of both trials.

Reviewer 3: ```suggestion
We will not know how effective vitamin C is as a therapeutic for quite some time due to the length of both trials.
```

> However, the trials will provide crucial evidence on the efficacy of vitamin C as a therapeutic for COVID-19 infection. 

Reviewer 3: ```suggestion
When completed, the trials will provide crucial evidence on the efficacy of vitamin C as a therapeutic for COVID-19 infection. 
```

###### Summary

> The evidence suggests that vitamin C supplementation can shorten the duration of a cold, reduce an individual's susceptibility to infections, and shorten a patient's stay in ICU when administered at high doses.

Reviewer 3: ```suggestion
Some evidence suggests that vitamin C supplementation can shorten the duration of a cold, reduce an individual's susceptibility to infections, and shorten a patient's stay in an ICU when administered at high doses.
```

> This is yet to be determined for COVID-19 infections. 

Reviewer 3: ```suggestion
We don't yet understand if these findings apply to COVID-19. 
```

> The ongoing trials in China and Italy along with with other potential trials intended in the United States and beyond will provide crucial information on the therapeutic value of vitamin C supplementation for COVID-19 infections.  

Reviewer 3: ```suggestion
There are ongoing trials in China and Italy that will inform our understanding of the therapeutic value of vitamin C supplementation for COVID-19.  
```

> The ongoing trials in China and Italy along with with other potential trials intended in the United States and beyond will provide crucial inforamtion on the therapeutic value of vitamin C supplementation for COVID-19 infections.  

Reviewer: ```suggestion
The ongoing trials in China and Italy along with with other potential trials intended in the United States and beyond will provide crucial information on the therapeutic value of vitamin C supplementation for COVID-19 infections.  
```
